ginga/pilw/ImageViewPil.py

```diff
@@ -4,13 +4,13 @@
 # This is open-source software licensed under a BSD license.
 # Please see the file LICENSE.txt for details.
 
-from io import BytesIO
```

Comment:
Note: This syntax is Python 3 only. Since Python 2 is still supported, I changed it to use `six` here and in some other relevant PRs as I see them. However, we should do a global sweep after these PEP8 PRs are merged to see if there are any remaining `from io import BytesIO`.

Reply:
EDIT: Nevermind... I misread the `six` documentation. Will revert this. Sorry for the confusion!